src/model/position.js

```diff
@@ -319,6 +319,27 @@ export default class Position {
 	}
 
 	/**
+	 * Returns an {@link module:engine/model/element~Element} or {@link module:engine/model/documentfragment~DocumentFragment}
+	 * which is a common ancestor for both positions. The {@link #root roots} of these two positions must be identical.
+	 *
+	 * @param {module:engine/model/position~Position} position The second position.
+	 * @returns {module:engine/model/element~Element|module:engine/model/documentfragment~DocumentFragment|null}
+	 */
+	getCommonAncestor( position ) {
+		if ( this.root !== position.root ) {
+			return null;
+		}
+
+		const node = this.root.getNodeByPath( this.getCommonPath( position ) );
+
+		if ( node.is( 'text' ) ) {
```

Comment:
This is something I expected from this method's model implementation. But I wonder what other think. Is "text node" a position's ancestor?